packages/notifications/project.json

```diff
@@ -47,6 +47,27 @@
         "commitMessageFormat": "release: bump {projectName} to {version} [skip ci]"
       }
     },
+    "integration-v2": {
```

Comment:
@catastrophe-brandon not blocking; but in the future I wonder if we can parameterize these to accept `v1` or `v2` etc to reduce duplication in the project.json and downstream. In reality I'm not sure how many versions of APIs we will really have at any given time, but I don't want this to get out of hand.